pkg/controller/controller.go

```diff
@@ -1199,3 +1205,664 @@ func (c *Controller) handleObject(obj interface{}) {
+	gatewayConfUserName := "wso2am-p2-am-gateway-conf-" + apimanager.Name
+	gatewayConfUser, err := c.configMapLister.ConfigMaps(apimanager.Namespace).Get(gatewayConfUserName)
+	if errors.IsNotFound(err) {
+		gatewayConfUser, err = c.kubeclientset.CoreV1().ConfigMaps(apimanager.Namespace).Create(pattern2.MakeConfigMap(apimanager, gatewayConfWso2))
+		if err != nil {
+			fmt.Println("Creating Gateway configmap in user specific ns", gatewayConfUser)
+		}
+	}
+
+	kmConfName := "wso2am-p2-am-km-conf"
+	kmConfWso2, err := c.configMapLister.ConfigMaps("wso2-system").Get(kmConfName)
+	kmConfUserName := "wso2-p2-am-km-conf" + apimanager.Name
+	kmConfUser, err := c.configMapLister.ConfigMaps(apimanager.Namespace).Get(kmConfUserName)
+	if errors.IsNotFound(err) {
+		kmConfUser, err = c.kubeclientset.CoreV1().ConfigMaps(apimanager.Namespace).Create(pattern2.MakeConfigMap(apimanager, kmConfWso2))
+		if err != nil {
+			fmt.Println("Creating Key Manager configmap in user specific ns", kmConfUser)
+		}
+	}
+
+	if enableAnalytics {
+		analyticsBinConfName := "wso2am-analytics-bin"
+		analyticsBinConfWso2, err := c.configMapLister.ConfigMaps("wso2-system").Get(analyticsBinConfName)
+		analyticsBinConfUserName := "wso2am-analytics-bin-" + apimanager.Name
+		analyticsBinConfUser, err := c.configMapLister.ConfigMaps(apimanager.Namespace).Get(analyticsBinConfUserName)
+		if errors.IsNotFound(err) {
+			analyticsBinConfUser, err = c.kubeclientset.CoreV1().ConfigMaps(apimanager.Namespace).Create(pattern2.MakeConfigMap(apimanager, analyticsBinConfWso2))
+			if err != nil {
+				fmt.Println("Creating analytics bin configmap in user specified ns", analyticsBinConfUser)
+			}
+		}
+
+	}
+
+	// Parse the object and look for it’s deployment
+	// Use a Lister to find the deployment object referred to in the Apimanager resource
+	// Get apim instance 1 deployment name using hardcoded value
+
+	pubDevTm1num := 0
+	pubDevTm2num := 0
+	gatewaynum := 0
+	kmnum := 0
+	dashnum := 0
+	worknum := 0
+
+	totalProfiles := len(apimanager.Spec.Profiles)
+
+	i := 0
+
+	if totalProfiles > 0 {
+		for i = 0; i < totalProfiles; i++ {
+			if apimanager.Spec.Profiles[i].Name == "api-pub-dev-tm-1" {
+				pubDevTm1num = i
+			}
+			if apimanager.Spec.Profiles[i].Name == "api-pub-dev-tm-2" {
+				pubDevTm2num = i
+			}
+			if apimanager.Spec.Profiles[i].Name == "analytics-dashboard" {
+				dashnum = i
+			}
+			if apimanager.Spec.Profiles[i].Name == "analytics-worker" {
+				worknum = i
+			}
+			if apimanager.Spec.Profiles[i].Name == "api-keymanager" {
+				kmnum = i
+			}
+			if apimanager.Spec.Profiles[i].Name == "api-gateway" {
+				gatewaynum = i
+			}
+		}
+	}
+
+	// Get mysql deployment name using hardcoded value
+	mysqldeployment, err := c.deploymentsLister.Deployments(apimanager.Namespace).Get(mysqldeploymentName)
+
+	if useMysqlPod {
+		// If the resource doesn't exist, we'll create it
+		if errors.IsNotFound(err) {
+			//y:= pattern1.AssignMysqlConfigMapValues(apimanager,configmap)
```

Comment:
sall we remove the comment if not important

Reply:
addressed via https://github.com/wso2/k8s-wso2am-operator/pull/89/commits/788038d4b86839a0ede2e666405f0c4f501c0dbf